doc/README.md

```diff
@@ -4,7 +4,7 @@ This directory contains documentation for FreeRADIUS.  The
 documentation is available under the Creative Commons Non-Commercial
 license, as given in the `LICENSE` file in this directory.
 
-[Introduction](introduction/) - Introduction to FreeRADIUS and RADIUS
+[Introduction](introduction/aaa) - Introduction to FreeRADIUS and RADIUS
```

Comment:
I would really prefer to leave the "main page" as just `introduction/`.  That allows the main page to have generic text, instead of being specific to AAA.

Reply:
@alandekok But, we don't have any README inside the directory. we 've only the `aaa.md` file there. so, maybe create an `index.html` into the introduction listing the content of directory? in this case, referring to `aaa`?

Reply:
The point is that we're fixing the documentation.  We're not just rearranging it.  This means adding documentation where it's missing.  In this case, a "main page" that serves as an introduction, and which points to the other documentation.

Reply:
All right. I made all the changes and you could see the current changes in https://www.jorgepereira.com.br/temp/freeradius-doc/